src/graphics/C4Shader.cpp

```diff
@@ -265,9 +265,9 @@ int C4Shader::ParsePosition(const char *szWhat, const char **ppPos)
 
 	// Lookup name
 	int iPosition = -1;
-	for (unsigned int i = 0; i < sizeof(C4SH_PosNames) / sizeof(*C4SH_PosNames); i++) {
-		if (SEqual(Name.getData(), C4SH_PosNames[i].Name)) {
-			iPosition = C4SH_PosNames[i].Position;
+	for (auto & C4SH_PosName : C4SH_PosNames) {
+		if (SEqual(Name.getData(), C4SH_PosName.Name)) {
+			iPosition = C4SH_PosName.Position;
```

Comment:
[reposting this here where it belongs because my email reply went to the PR's general comments]

Yeah, at one point I really want to change all of the ugly 90s hungarian
style to something more readable, maybe one file at a time. While we're
dreaming of a better world, I'd also like to unify function names and
parameter names and members and all the other stuff that's accumulated
over time.